src/service/performance-impl.js

```diff
@@ -517,6 +525,26 @@ export class Performance {
     }
   }
 
+  /**
+   * Tick both the eager and slow element ratios.
+   */
+  tickSlowElementRatio_() {
+    if (this.slowElementRatioTicked_) {
+      return;
+    }
+    if (!this.resources_) {
+      const TAG = 'Performance';
+      dev().error(TAG, 'Failed to tick ser due to null resources');
+      return;
+    }
+
+    this.slowElementRatioTicked_ = true;
+    this.tickDelta(
+      TickLabel.SLOW_ELEMENT_RATIO,
+      this.resources_.getSlowElementRatio()
+    );
```

Comment:
Do we need to flush here?

Reply:
Ah, I didn't realize this. Just added it. 
Also my gut says we have too many flushes. Could probably optimize by combining the flush for `cls`, `ser`, and `lcp`.